app/models/task.rb

```diff
@@ -264,6 +264,34 @@ def do_hold(last_check)
     end
   end
 
+  def name
```

Comment:
I agree they are quite similar. I made my own because the other method returns "TurnOn" without a space (as opposed to "Turn On"), which I thought looked awkward on the chart. Also, if it is applied to a thermostat, the label on the graph would be "RunThermostat Thermostat 4" as opposed to "Run Thermostat 4", so this method got rid of the double Thermostat.